controller_manager/controller_manager/controller_manager_services.py

```diff
@@ -26,6 +26,29 @@
 )
 
 import rclpy
+import yaml
+from rcl_interfaces.msg import Parameter
+
+# @note: The versions conditioning is added here to support the source-compatibility with Humble
+# The `get_parameter_value` function is moved to `rclpy.parameter` module from `ros2param.api` module from version 3.6.0
+try:
+    from rclpy.parameter import get_parameter_value
+except ImportError:
+    from ros2param.api import get_parameter_value
+from ros2param.api import call_set_parameters
+
+
+# from https://stackoverflow.com/a/287944
+class bcolors:
+    HEADER = "\033[95m"
+    OKBLUE = "\033[94m"
+    OKCYAN = "\033[96m"
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    ENDC = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
```

Comment:
Please see this only as a comment, not as a request:

This might be considered to be moved to a different file? Importing this from the controller_manager_services module seems to work fine for the given use cases, but maybe it would be more intuitive to declare it in the __init__ file directly or move it to a separate helpers module.

Reply:
At the very beginning i wanted to do it in the __init__ file itself, but then if that's the case I couldn't use it inside the controller_manager_services file and then later add it as a module inside the __init__ file. So, in theory you should be able to import in other python files as `from controller_manager import bcolors`

Reply:
And besides that, we should rethink on renaming colors, the names are quite fuzzy.

Reply:
Sure! Good idea. Maybe we can do it in a separate PR? What do you say?

Reply:
renaming separately